src/screens/RecipeDetails/components/Header/styles.ts

```diff
@@ -6,13 +6,18 @@ export const styles = StyleSheet.create({
   header: {
     position: 'absolute',
     flexDirection: 'row',
+    justifyContent: 'flex-end',
+    alignItems: 'flex-end',
+    alignContent: 'flex-end',
```

Comment:
https://reactnative.dev/docs/flexbox#align-content
"This only has effect when items are wrapped to multiple lines using flexWrap."